cargo-dist/tests/snapshots/long-help.snap

```diff
@@ -18,6 +18,7 @@ Commands:
   manifest  Generate the final build manifest without running any builds
   plan      Get a plan of what to build (and check project status)
   host      Host artifacts
+  update    Performs a self-update, if a new version is available
```

Comment:
given what we may do in the future do we want this to be "self update" ?

Reply:
Seems reasonable!